cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -510,38 +509,48 @@ def urls(self, key, value):
     sub_u = clean_val("u", value, str, req=True)
 
     eitems_ebl = self["_migration"]["eitems_ebl"]
+    eitems_safari = self["_migration"]["eitems_safari"]
     eitems_external = self["_migration"]["eitems_external"]
     eitems_proxy = self["_migration"]["eitems_proxy"]
     eitems_files = self["_migration"]["eitems_file_links"]
 
-    url = {"value": sub_u}
+    eitem = {"url": {"value": sub_u}, "open_access": False}
```

Comment:
I wouldn't name it eitem, because it is not this record type yet, it is just a dict representing the eitem's url. It looks confusing when you return eitem["url"] in the end.
From what I have understood from the rest of the code, adding the open_access here is not necessary, since you do it based on the provider (EBL/SAFARI/FILE/PROXY/OTHERS) and you also rely on the `['_migration']['eitems_open_access']` to solve the problem for eitems_external